apps/document-check-function/check-new-document/index.js

```diff
@@ -0,0 +1,32 @@
+import { sendDocumentStateAction } from './back-office-api-client.js';
+import { scanStream } from './clam-av-client.js';
+
+/**
+ * @param {boolean} isInfected
+ * @returns {string}
+ */
+const mapIsInfectedToMachineAction = (isInfected) => {
+	return isInfected ? 'check_fail' : 'check_pass';
+};
+
+/**
+ * @param {string} blobUri
+ * @returns {{caseId: string, guid: string}}
+ */
+const getBlobCaseIdAndGuid = (blobUri) => {
+	const uriParts = blobUri.split('/');
+
+	return { caseId: uriParts[1], guid: uriParts[2] };
```

Comment:
Might be worth checking the length of uriParts here and logging the URI with an error if something goes wrong

Reply:
Good shout. Adding something now